docs/hooks/encounter-start.md

```diff
@@ -0,0 +1,40 @@
+# `encounter-start`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 1.0
+| hookMaturity | 0 - Draft 
+
+## Workflow
+
+This hook is invoked when the user is initiating a new encounter.  In an inpatient setting, this would be the time of admission.  In an outpatient/community environment, this would be the time of patient-check-in for a face-to-face or equivalent for a virtual/telephone encounter.  The Encounter should either be in one of the following states: `planned` | `arrived` | `triaged` | `in-progress`.  Note that there can be multiple 'starts' for the same encounter as each user becomes engaged.  For example, when a scheduled encounter is presented at the beginning of the day for planning purposes, when the patient arrives, when the patient first encounters a clinician, etc.  Hooks may present different information depending on user role and Encounter.status.
+
+Note: This is distinct from the `patient-view` hook which occurs any time the patient's record is looked at - which might be done outside the context of any encounter and will often occur during workflows that are not linked to the initiation of an encounter.
```

Comment:
This is super helpful in terms of understanding the thought process here. For service developers, do we have examples/thoughts about when they'd want to support *only* `encoutner-start-` and *not* also support `patient-view`? May recommendations that make sense in the context of deciding what to do during the current encounter might also be reasonable in the context of planning for a future encounter.

Reply:
If a pre-authorization is needed for a service, it makes sense to show it at time of admission, but not every time a nurse or physician looks at a record during a week-long stay.  The same would be true if there are diagnostic tests the patient is due for or steps of a clinical trial that need to be done during a scheduled encounter.  The primary purpose of the separation is to help keep hooks from being overwhelming or noisy.  If the intention is "show them once, let them make a decision and don't bug them again", enounter-start is appropriate.  If the intention is keep bugging the clinician even if the patient is no longer in their care until a record is updated, then patient-view makes sense.

Reply:
This generally makes sense, and is good flavor.

If you've got a pre-authjorization step required and someone *doesn't* actually complete it at admission time, do you want it to just disappear from the virtual todo list?

Also would note that a given CDS Service can track which users it has prompted, etc, and could decide to stop showing cards.

Anyway, I'm not challenging the utility of this hook; I just want to do as much as we can to help articulate when each hook is more appropriate, and when a developer would want to support both.